api/config/config_test.json

```diff
@@ -133,5 +133,9 @@
   "reporting": {
     "dataset": "reporting_test",
     "maxRowsPerInsert": 2500
+  },
+  "ras": {
+    "host": "https:\/\/stsstg.nih.gov",
```

Comment:
both are RAS staging. They only have one instance. And they created registered us with one client ID and secret, but with different redirect URL. (aou-test/ras-callback vs localhost:4200/ras-callback)